HARK/ConsumptionSaving/tests/test_IndShockConsumerType.py

```diff
@@ -94,8 +95,9 @@ def test_simulated_values(self):
         self.agent.simulate()
 
         self.assertAlmostEqual(self.agent.MPCnow[1], 0.5711503906043797)
```

Comment:
I think this test might have to go too. MPC now depends on assets now which are stochastic

Reply:
good to know, thanks